gearcatalog/frontend/babel.config.js

```diff
@@ -4,4 +4,8 @@ module.exports = {
     ['@babel/preset-react'],
     ['@babel/preset-typescript'],
   ],
+  plugins: [
+    // TODO:#1 Remove after converting GearList component to functional
```

Comment:
Miscommunication here. The number should be the Github Issue number. So for this it should be #3. This way people can look it up the ticket easily by typing the issue number directly into Github. The issue names don't need to have "TODO #X" in the titles.

Reply:
Ok, Should be fixed not.